pkg/api/convertertov20180930preview.go

```diff
@@ -40,6 +40,10 @@ func ConvertToV20180930preview(cs *OpenShiftManagedCluster) *v20180930preview.Op
 		VnetCIDR:   &cs.Properties.NetworkProfile.VnetCIDR,
 		PeerVnetID: cs.Properties.NetworkProfile.PeerVnetID,
 	}
+	oc.Properties.AuthGroupSync = &v20180930preview.AuthGroupSync{
+		Admins:  &cs.Properties.AuthGroupSync.Admins,
```

Comment:
You should be returning these only when they are non-empty strings.

Reply:
ok